### PR TITLE
Set ad configuration values if using explicit placement

### PR DESCRIPTION
**Pull request by davidfischer**
This only applies if the project is setting an explicit placement. This sets the values on the ad element sanely.

Ref: https://github.com/readthedocs/readthedocs.org/pull/7623#issuecomment-719730738

### CODE REVIEW

File: readthedocs/core/static-src/core/js/doc-embed/sponsorship.js [hunk 1]
```diff
@@ -32,6 +32,11 @@ function create_ad_placement() {
     var offset;
 
     if ($(EXPLICIT_PLACEMENT_SELECTOR).length > 0) {
+        $(EXPLICIT_PLACEMENT_SELECTOR).attr("data-ea-publisher", "readthedocs");
```
Comment (ericholscher) [inline readthedocs/core/static-src/core/js/doc-embed/sponsorship.js:35]: Do we not need this to respect the value coming back from the sustainability API?

**Review by ericholscher — APPROVED** [review]

File: readthedocs/core/static-src/core/js/doc-embed/sponsorship.js (hunk 1 above)
Comment (davidfischer) [reply readthedocs/core/static-src/core/js/doc-embed/sponsorship.js:35]: That will set it later.

Comment (ericholscher) [reply readthedocs/core/static-src/core/js/doc-embed/sponsorship.js:35]: Ah, reading through the code, sounds like that's set dynamically after the call returns.

Comment (ericholscher) [reply readthedocs/core/static-src/core/js/doc-embed/sponsorship.js:35]: Cool, should get this merged for deploy 👍

**Pull request merged**